workday.sgnl.yaml

```diff
@@ -0,0 +1,95 @@
+description: "Workday as a System of Record"
+address: "wd2-impl-services1.workday.com" 
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: Workday-1.0.0
+adapterConfig: ewogICAgImFwaVZlcnNpb24iOiAidjEiLAogICAgIm9yZ2FuaXphdGlvbklkIjogInt7SW5wdXQgUmVxdWlyZWR9fSIKfQ==
+auth:
+  - bearer:
+      authToken: "Bearer {{Input Required}}" # Access token
+entities:
+  Worker:
+    displayName: Worker
+    externalId: allWorkers # Name of the table in Workday
+    description: Entity representing a Worker in Workday
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: workerId
+        externalId: $.worker.id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: workerDescriptor
+        externalId: $.worker.descriptor
+        type: String
+      - name: employeeID
```

Comment:
this is the only other one I'd suggest indexing - it will likely be used in relationships.

Reply:
added